src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -137,17 +133,16 @@ the MIT License. See LICENSE in the project root for license information. -->
                         </local:SettingContainer>
 
                         <!--Hidden-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}"
-                                          Visibility="{x:Bind State.Profile.IsBaseLayer, Mode=OneWay, Converter={StaticResource InvertedBooleanToVisibilityConverter}}">
-                            <ToggleSwitch x:Uid="Profile_Hidden"
-                                          IsOn="{x:Bind State.Profile.Hidden, Mode=TwoWay}"/>
-                        </ContentPresenter>
+                        <local:SettingContainer x:Uid="Profile_Hidden"
+                                                Visibility="{x:Bind State.Profile.IsBaseLayer, Mode=OneWay, Converter={StaticResource InvertedBooleanToVisibilityConverter}}">
+                            <ToggleSwitch IsOn="{x:Bind State.Profile.Hidden, Mode=TwoWay}"/>
+                        </local:SettingContainer>
 
                         <!--Delete Button-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                            <StackPanel Visibility="{x:Bind State.Profile.IsBaseLayer, Mode=OneWay, Converter={StaticResource InvertedBooleanToVisibilityConverter}}">
+                        <local:SettingContainer Visibility="{x:Bind State.Profile.IsBaseLayer, Mode=OneWay, Converter={StaticResource InvertedBooleanToVisibilityConverter}}">
+                            <StackPanel>
                                 <Button IsEnabled="{x:Bind State.Profile.CanDeleteProfile}"
-                                    Style="{StaticResource DeleteButtonStyle}">
+                                        Style="{StaticResource DeleteButtonStyle}">
                                     <Button.Resources>
                                         <ResourceDictionary>
                                             <ResourceDictionary.ThemeDictionaries>

```

Comment:
this part is weird to me -- it's _not_ a setting.

Reply:
even if it were inheritable, it cannot be overridden or reset

Reply:
Fair enough. Looks like we were just using this for the `StandardControlMargin`, so I added that in.